runtime/runtime-params-estimator/src/wasmer_estimator.rs

```diff
@@ -61,14 +62,32 @@ fn call() -> (Option<VMOutcome>, Option<VMError>) {
 const NUM_ITERATIONS: u64 = 10;
 
 /// Cost of the most CPU demanding operation.
-pub fn cost_per_op(gas_metric: GasMetric) -> u64 {
+pub fn cost_per_op(gas_metric: GasMetric) -> Ratio<u64> {
     // Call once for the warmup.
     let (outcome, _) = call();
     let outcome = outcome.unwrap();
     let start = start_count(gas_metric);
     for _ in 0..NUM_ITERATIONS {
         call();
     }
-    let cost = end_count(gas_metric, &start);
-    cost / (outcome.burnt_gas * NUM_ITERATIONS)
+    let measured = end_count(gas_metric, &start);
+    // Those proportions led to the used formula:
+    //   outcome.burnt_gas = gas_burned(call)
```

Comment:
Not sure how to read this. I suggest to clarify what is given and what is derived. To my understanding `outcome.burnt_gas` is a given, so they way to interpret `outcome.burnt_gas = gas_burned(call)` would be -- "this is how it is computed".

Reply:
Rewrote.